src/Device/Driver/XCVario.cpp

```diff
@@ -31,6 +31,7 @@ Copyright_License {
 #include "NMEA/InputLine.hpp"
 #include "util/Clamp.hpp"
 #include "Atmosphere/Pressure.hpp"
+#include "LogFile.hpp"
```

Comment:
Please ... no more useless log messages.

Reply:
just only one log in an error condition